### PR TITLE
Ancient europe

### PR DESCRIPTION
**Pull request by AliPearson**
Added demographic model of Ancient Europe.

### CODE REVIEW

File: stdpopsim/catalog/HomSap/demographic_models.py [hunk 1]
```diff
@@ -1548,3 +1548,136 @@ def _ooa_4pop():
 
 
 _species.add_demographic_model(_ooa_4pop())
+
+
+def _ancient_europe():
+    id = "AncientEurope_4P21"
```
Comment (grahamgower) [inline stdpopsim/catalog/HomSap/demographic_models.py:1554]: The naming convention for model IDs is to use the citation's author initial in the last bit of the ID. Below, you've suggested the primary author is (will be?) Allentoft, so the ID should be AncientEurope_4A21.

File: stdpopsim/catalog/HomSap/demographic_models.py [hunk 2]
```diff
@@ -1548,3 +1548,136 @@ def _ooa_4pop():
...
+        ),
+        stdpopsim.Population(id="Pop1", description="Yamnaya/CHG"),
+        stdpopsim.Population(id="Pop2", description="WHG/NorthernEuropean"),
+        stdpopsim.Population(id="Pop3", description="EHG"),
+    ]
+    citations = [
+        stdpopsim.Citation(
+            author="Allentoft et al.",
+            year="in progress",
+            doi="in progress",
```
Comment (grahamgower) [inline stdpopsim/catalog/HomSap/demographic_models.py:1582]: Our defacto policy for model inclusion is that the model appears in a publication. Do you have an idea of when the manuscript is expected to appear? I think a preprint ought to be fine for inclusion here.

**Review by grahamgower — COMMENTED** [review]
This looks good @AliPearson, thanks! I left a couple of comments/questions below.

File: stdpopsim/catalog/HomSap/demographic_models.py (hunk 2 above)
Comment (petrelharp) [reply stdpopsim/catalog/HomSap/demographic_models.py:1582]: preprint definitely counts as "publication".

File: docs/parameter_tables/HomSap/AncientEurope_4P21.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
+Population size,"50,000",Bronze Age pop. size
+Population size,"5,000",Yamnaya pop. size
+Population size,"10,000",Western Hunter-Gatherer pop. size
+Population size,"10,000,Eastern Hunter-Gatherer pop. size
```
Comment (petrelharp) [inline docs/parameter_tables/HomSap/AncientEurope_4P21.csv:4]: missing a closing parenthesis here (I think this is the cause of the CI failure)

**Comment by petrelharp** [comment]
Just checking if you know what to do next, @AliPearson? Either fix up the things mentioned above or discuss them, then push a new commit; hopefully that will fix the failing check (the docs build) as well. If all is good then, you'll then squash & rebsae and we'll merge! Let us know if you need more direction than that. Thanks, this looks great!

**Comment by AliPearson** [comment]
Sorry to be slow! I’ll make those changes and push it asap

**Comment by petrelharp** [comment]
No worries - just checking in!

**Comment by grahamgower** [comment]
Yep, this looks like it's almost ready to go @AliPearson, I think we're just waiting for something citable (which can be a preprint)?

**Comment by grahamgower** [comment]
I think it's the recent Allentoft et al. manuscript: https://www.biorxiv.org/content/10.1101/2022.05.04.490594v2
Can you confirm @AliPearson?

**Comment by AliPearson** [comment]
Hi. Sorry for the delay! Yes it’s the recent Allentoft manuscript.

On 11 May 2022, at 08:20, Graham Gower ***@***.******@***.***>> wrote:



I think it's the recent Allentoft et al. manuscript: https://www.biorxiv.org/content/10.1101/2022.05.04.490594v2<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.biorxiv.org%2Fcontent%2F10.1101%2F2022.05.04.490594v2&data=05%7C01%7Cap885%40universityofcambridgecloud.onmicrosoft.com%7C3490070150ce4d23852008da331ed83d%7C49a50445bdfa4b79ade3547b4f3986e9%7C0%7C0%7C637878504811351268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LkkHs8T7vh5xEDWDG%2FQDzjIbf2mTapGcSbP%2FheSzwS0%3D&reserved=0>
Can you confirm @AliPearson<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAliPearson&data=05%7C01%7Cap885%40universityofcambridgecloud.onmicrosoft.com%7C3490070150ce4d23852008da331ed83d%7C49a50445bdfa4b79ade3547b4f3986e9%7C0%7C0%7C637878504811351268%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dSNXYcpX0FZvG%2BBBkRr7eE4WhxiY%2BawNTbhHoouBiW4%3D&reserved=0>?

**Comment by petrelharp** [comment]
Ok - the rebase was a bit painful, but I got it sorted. I've updated your repository (since the one this PR points to), @AliPearson, in case you're not familiar with this, you'll have to do something like `git checkout AncientEurope; git fetch; git reset --hard github/AncientEurope` (where maybe it's `origin` instead of `github`?) - let me know if anything is amiss. And, I've got a backup of your original branch locally, in case we need to go back to it, but I'm pretty sure all your changes are in here.

**Comment by petrelharp** [comment]
I've confirmed with @AliPearson that this is good to go, so I'll merge this and open the QC issue.

**Pull request merged**